charts/sonarqube-dce/README.md

```diff
@@ -72,6 +72,16 @@ $ helm delete kindly-newt
 
 Please read the official documentation prerequisites [here](https://docs.sonarqube.org/latest/requirements/prerequisites-and-overview/).
 
+### Kubernetes - Pod Security Standards
+
+Pod Security Standards
+
+The following Pod Security levels cannot be used in combination with SonarQube's chart:
+
+* Baseline. The "init-sysctl" and "init-fs" containers requires securityContext.privileged=true.
```

Comment:
Given that we are using markdown here, could you please replace all occurrences of double quotes with single ones? This might help the readability :)

"init-fs" -> `init-fs`